kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyConfigSecret.java

```diff
@@ -43,33 +62,56 @@ static String secretName(KafkaProxy primary) {
     @Override
     protected Secret desired(KafkaProxy primary,
                              Context<KafkaProxy> context) {
-
+        // formatter=off
```

Comment:
I don't see a `// formatter=on` 

Which maybe isn't a big deal but does feel a bit of a let down.

Reply:
The regex also turns the formatting back on at the next `return`. I guess it's an open question whether that's good (if you forget to turn it on then it happens anyway) or bad (it's not explicit).

Reply:
I can live with it. 

One other thing I noticed intelliJ defaults to `@formatter:off` & `@formatter:on` do we want to match that or are we happy with it and just want to shoot the one maven applies?

Reply:
Let's fit in with intellij.